docs/templates/snakemake/README.md

```diff
@@ -0,0 +1,59 @@
+# {Snakemake pipeline template}
+
+## How to create your snakemake pipeline for APAeval
+Read this section, but do NOT include it your final README.
+> * Copy the whole directory `snakemake` with its' subfolders to suitable location.
+> * Rename `snakemake` to `[METHOD]_snakemake`, [METHOD] being the name of the tool you're building a workflow for.
+> * Adjust names and contents of files that appear in this [NOTATION].
+> * Place any scripts or subworkflows you're going to use into the directories `workflow/scripts` and `workflow/rules`, respectively.
+> * Write your workflow in `Snakefile`.
+> * Here are the [snakemake docs](https://snakemake.readthedocs.io/en/stable/index.html)
+> * General good practices:
+>     * Test your code with [snakemake --lint](https://snakemake.readthedocs.io/en/stable/snakefiles/writing_snakefiles.html#best-practices).
```

Comment:
Specify which Snakemake version people should use

Reply:
Addressed in `snakemake.yaml`